src/__tests__/restLink.ts

```diff
@@ -940,13 +940,13 @@ describe('Can customize/parse the response before passing to Apollo', () => {
 
       const link = new RestLink({
         // This is purpsefully wrong so that we verify the endpoint one is called
-        responseTransformer: data => data.collection,
+        responseTransformer: async data => data.collection,
```

Comment:
This line confused me at first until I noticed the line at the top. I know this isn't your code originally, but could we change this to throw an error in this case?

```suggestion
        responseTransformer: async data => throw new Error('Should have called endpoint responseTransformer'),
```

Reply:
LGTM but then the comment becomes redundant and should be removed. Also I don't think the async should be there, my bad.